<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1192</width>
    <height>894</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QLabel" name="labelPic">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>384</width>
     <height>384</height>
    </rect>
   </property>
   <property name="text">
    <string>Please insert image.</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelPri">
   <property name="geometry">
    <rect>
     <x>404</x>
     <y>50</y>
     <width>384</width>
     <height>384</height>
    </rect>
   </property>
   <property name="text">
    <string>Waiting to pridict.</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelGt">
   <property name="geometry">
    <rect>
     <x>798</x>
     <y>50</y>
     <width>384</width>
     <height>384</height>
    </rect>
   </property>
   <property name="text">
    <string>Try to insert Ground truth.</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonPic">
   <property name="geometry">
    <rect>
     <x>142</x>
     <y>470</y>
     <width>120</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Choose Pic</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonPri">
   <property name="geometry">
    <rect>
     <x>536</x>
     <y>470</y>
     <width>120</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Pridict</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonGt">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>470</y>
     <width>120</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Choose Gt</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>400</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <underline>true</underline>
    </font>
   </property>
   <property name="text">
    <string>图片展示大小统一为384x384,根据原图大小进行等比缩放</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonCmiou">
   <property name="geometry">
    <rect>
     <x>142</x>
     <y>550</y>
     <width>120</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Count Miou</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidgetClass">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>590</y>
     <width>600</width>
     <height>90</height>
    </rect>
   </property>
   <property name="baseSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="rowCount">
    <number>3</number>
   </property>
   <property name="columnCount">
    <number>6</number>
   </property>
   <attribute name="horizontalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <row/>
   <row/>
   <row/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <item row="0" column="0">
    <property name="text">
     <string>水面</string>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
    <property name="foreground">
     <brush brushstyle="SolidPattern">
      <color alpha="255">
       <red>0</red>
       <green>0</green>
       <blue>0</blue>
      </color>
     </brush>
    </property>
    <property name="flags">
     <set>NoItemFlags</set>
    </property>
   </item>
   <item row="0" column="1">
    <property name="text">
     <string>绿树</string>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
    <property name="foreground">
     <brush brushstyle="SolidPattern">
      <color alpha="255">
       <red>0</red>
       <green>0</green>
       <blue>0</blue>
      </color>
     </brush>
    </property>
    <property name="flags">
     <set>NoItemFlags</set>
    </property>
   </item>
   <item row="0" column="2">
    <property name="text">
     <string>低植被</string>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
    <property name="foreground">
     <brush brushstyle="SolidPattern">
      <color alpha="255">
       <red>0</red>
       <green>0</green>
       <blue>0</blue>
      </color>
     </brush>
    </property>
    <property name="flags">
     <set>NoItemFlags</set>
    </property>
   </item>
   <item row="0" column="3">
    <property name="text">
     <string>建筑</string>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
    <property name="foreground">
     <brush brushstyle="SolidPattern">
      <color alpha="255">
       <red>0</red>
       <green>0</green>
       <blue>0</blue>
      </color>
     </brush>
    </property>
    <property name="flags">
     <set>NoItemFlags</set>
    </property>
   </item>
   <item row="0" column="4">
    <property name="text">
     <string>车辆</string>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
    <property name="foreground">
     <brush brushstyle="SolidPattern">
      <color alpha="255">
       <red>0</red>
       <green>0</green>
       <blue>0</blue>
      </color>
     </brush>
    </property>
    <property name="flags">
     <set>NoItemFlags</set>
    </property>
   </item>
   <item row="0" column="5">
    <property name="text">
     <string>其他</string>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
    <property name="foreground">
     <brush brushstyle="SolidPattern">
      <color alpha="255">
       <red>0</red>
       <green>0</green>
       <blue>0</blue>
      </color>
     </brush>
    </property>
    <property name="flags">
     <set>NoItemFlags</set>
    </property>
   </item>
   <item row="1" column="0">
    <property name="text">
     <string/>
    </property>
    <property name="background">
     <brush brushstyle="SolidPattern">
      <color alpha="255">
       <red>255</red>
       <green>255</green>
       <blue>255</blue>
      </color>
     </brush>
    </property>
    <property name="flags">
     <set>NoItemFlags</set>
    </property>
   </item>
   <item row="1" column="1">
    <property name="text">
     <string/>
    </property>
    <property name="background">
     <brush brushstyle="SolidPattern">
      <color alpha="255">
       <red>0</red>
       <green>255</green>
       <blue>0</blue>
      </color>
     </brush>
    </property>
    <property name="flags">
     <set>NoItemFlags</set>
    </property>
   </item>
   <item row="1" column="2">
    <property name="text">
     <string/>
    </property>
    <property name="background">
     <brush brushstyle="SolidPattern">
      <color alpha="255">
       <red>0</red>
       <green>255</green>
       <blue>255</blue>
      </color>
     </brush>
    </property>
    <property name="flags">
     <set>NoItemFlags</set>
    </property>
   </item>
   <item row="1" column="3">
    <property name="text">
     <string/>
    </property>
    <property name="background">
     <brush brushstyle="SolidPattern">
      <color alpha="255">
       <red>0</red>
       <green>0</green>
       <blue>255</blue>
      </color>
     </brush>
    </property>
    <property name="flags">
     <set>NoItemFlags</set>
    </property>
   </item>
   <item row="1" column="4">
    <property name="text">
     <string/>
    </property>
    <property name="background">
     <brush brushstyle="SolidPattern">
      <color alpha="255">
       <red>255</red>
       <green>255</green>
       <blue>0</blue>
      </color>
     </brush>
    </property>
    <property name="flags">
     <set>NoItemFlags</set>
    </property>
   </item>
   <item row="1" column="5">
    <property name="text">
     <string/>
    </property>
    <property name="background">
     <brush brushstyle="SolidPattern">
      <color alpha="255">
       <red>255</red>
       <green>0</green>
       <blue>0</blue>
      </color>
     </brush>
    </property>
    <property name="flags">
     <set>NoItemFlags</set>
    </property>
   </item>
   <item row="2" column="0">
    <property name="text">
     <string/>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
    <property name="foreground">
     <brush brushstyle="SolidPattern">
      <color alpha="255">
       <red>0</red>
       <green>0</green>
       <blue>0</blue>
      </color>
     </brush>
    </property>
    <property name="flags">
     <set>NoItemFlags</set>
    </property>
   </item>
   <item row="2" column="1">
    <property name="text">
     <string/>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
    <property name="foreground">
     <brush brushstyle="SolidPattern">
      <color alpha="255">
       <red>0</red>
       <green>0</green>
       <blue>0</blue>
      </color>
     </brush>
    </property>
    <property name="flags">
     <set>NoItemFlags</set>
    </property>
   </item>
   <item row="2" column="2">
    <property name="text">
     <string/>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
    <property name="foreground">
     <brush brushstyle="SolidPattern">
      <color alpha="255">
       <red>0</red>
       <green>0</green>
       <blue>0</blue>
      </color>
     </brush>
    </property>
    <property name="flags">
     <set>NoItemFlags</set>
    </property>
   </item>
   <item row="2" column="3">
    <property name="text">
     <string/>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
    <property name="foreground">
     <brush brushstyle="SolidPattern">
      <color alpha="255">
       <red>0</red>
       <green>0</green>
       <blue>0</blue>
      </color>
     </brush>
    </property>
    <property name="flags">
     <set>NoItemFlags</set>
    </property>
   </item>
   <item row="2" column="4">
    <property name="text">
     <string/>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
    <property name="flags">
     <set>NoItemFlags</set>
    </property>
   </item>
   <item row="2" column="5">
    <property name="text">
     <string/>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
    <property name="foreground">
     <brush brushstyle="SolidPattern">
      <color alpha="255">
       <red>0</red>
       <green>0</green>
       <blue>0</blue>
      </color>
     </brush>
    </property>
    <property name="flags">
     <set>NoItemFlags</set>
    </property>
   </item>
  </widget>
  <widget class="QTableWidget" name="tableWidgetMiou">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>520</y>
     <width>500</width>
     <height>60</height>
    </rect>
   </property>
   <property name="showGrid">
    <bool>true</bool>
   </property>
   <property name="gridStyle">
    <enum>Qt::SolidLine</enum>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
   <property name="cornerButtonEnabled">
    <bool>true</bool>
   </property>
   <property name="rowCount">
    <number>2</number>
   </property>
   <property name="columnCount">
    <number>5</number>
   </property>
   <attribute name="horizontalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <row/>
   <row/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <item row="0" column="0">
    <property name="text">
     <string>Miou</string>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
    <property name="foreground">
     <brush brushstyle="SolidPattern">
      <color alpha="255">
       <red>0</red>
       <green>0</green>
       <blue>0</blue>
      </color>
     </brush>
    </property>
    <property name="flags">
     <set>ItemIsSelectable</set>
    </property>
   </item>
   <item row="0" column="1">
    <property name="text">
     <string>MPA</string>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
    <property name="foreground">
     <brush brushstyle="SolidPattern">
      <color alpha="255">
       <red>0</red>
       <green>0</green>
       <blue>0</blue>
      </color>
     </brush>
    </property>
    <property name="flags">
     <set>NoItemFlags</set>
    </property>
   </item>
   <item row="0" column="2">
    <property name="text">
     <string>Pricision</string>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
    <property name="background">
     <brush brushstyle="NoBrush">
      <color alpha="255">
       <red>0</red>
       <green>0</green>
       <blue>0</blue>
      </color>
     </brush>
    </property>
    <property name="foreground">
     <brush brushstyle="SolidPattern">
      <color alpha="255">
       <red>0</red>
       <green>0</green>
       <blue>0</blue>
      </color>
     </brush>
    </property>
    <property name="flags">
     <set>ItemIsSelectable</set>
    </property>
   </item>
   <item row="0" column="3">
    <property name="text">
     <string>Recall</string>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
    <property name="foreground">
     <brush brushstyle="SolidPattern">
      <color alpha="255">
       <red>0</red>
       <green>0</green>
       <blue>0</blue>
      </color>
     </brush>
    </property>
    <property name="flags">
     <set>NoItemFlags</set>
    </property>
   </item>
   <item row="0" column="4">
    <property name="text">
     <string>F1-Score</string>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
    <property name="foreground">
     <brush brushstyle="SolidPattern">
      <color alpha="255">
       <red>0</red>
       <green>0</green>
       <blue>0</blue>
      </color>
     </brush>
    </property>
    <property name="flags">
     <set>NoItemFlags</set>
    </property>
   </item>
   <item row="1" column="0">
    <property name="text">
     <string/>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
    <property name="foreground">
     <brush brushstyle="SolidPattern">
      <color alpha="255">
       <red>0</red>
       <green>0</green>
       <blue>0</blue>
      </color>
     </brush>
    </property>
    <property name="flags">
     <set>NoItemFlags</set>
    </property>
   </item>
   <item row="1" column="1">
    <property name="text">
     <string/>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
    <property name="flags">
     <set>NoItemFlags</set>
    </property>
   </item>
   <item row="1" column="2">
    <property name="text">
     <string/>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
    <property name="flags">
     <set>NoItemFlags</set>
    </property>
   </item>
   <item row="1" column="3">
    <property name="text">
     <string/>
    </property>
    <property name="flags">
     <set>NoItemFlags</set>
    </property>
   </item>
   <item row="1" column="4">
    <property name="text">
     <string/>
    </property>
    <property name="flags">
     <set>NoItemFlags</set>
    </property>
   </item>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
